EIPS/eip-7756.md

```diff
@@ -154,7 +155,7 @@ This is the trace output from the Ethereum Execution Specification Test from one
 {"pc":13,"op":90,"gas":"0x2fa9e66","gasCost":"0x2","memSize":0,"stack":["0x0","0x0","0x0","0x0","0x0","0x1000"],"depth":1,"refund":0,"opName":"GAS"}
 {"pc":14,"op":241,"gas":"0x2fa9e64","gasCost":"0x2eeb414","memSize":0,"stack":["0x0","0x0","0x0","0x0","0x0","0x1000","0x2fa9e64"],"depth":1,"refund":0,"opName":"CALL"}
 {"pc":0,"section":0,"op":227,"immediate":"0x0001","gas":"0x2eea9ec","gasCost":"0x5","memSize":0,"stack":[],"depth":2,"refund":0,"opName":"CALLF"}
```

Comment:
Ah interesting, I just noticed: so if `functionDepth` is `0` it is omitted in this trace. I think `functionDepth` 0 should be added here? Also to the STOP operation.

Reply:
`functionDepth` is optional when zero.  I'll add it to the spec.  Generally empty/zero/zero length is an optional member. Attempting to reduce repetition.